cmd/milmove/migrate.go

```diff
@@ -96,31 +120,131 @@ func migrateFunction(cmd *cobra.Command, args []string) error {
 
 	err = checkMigrateConfig(v, logger)
 	if err != nil {
-		logger.Fatal("invalid configuration", zap.Error(err))
+		return errors.Wrap(err, "invalid configuration")
 	}
 
 	if v.GetBool(cli.DbDebugFlag) {
 		pop.Debug = true
 	}
 
+	migrationPath := expandPaths(strings.Split(v.GetString(cli.MigrationPathFlag), ";"))
+	logger.Info(fmt.Sprintf("using migration path %q", migrationPath))
+
+	s3Migrations := false
+	for _, p := range migrationPath {
+		if strings.HasPrefix(p, "s3://") {
+			s3Migrations = true
+			break
+		}
+	}
+
+	var session *awssession.Session
+	if v.GetBool(cli.DbIamFlag) || s3Migrations {
+		c, errorConfig := cli.GetAWSConfig(v, v.GetBool(cli.VerboseFlag))
```

Comment:
Can we use full names instead of single character names?

```suggestion
		config, errorConfig := cli.GetAWSConfig(v, v.GetBool(cli.VerboseFlag))
```